README.md

```diff
@@ -63,7 +63,7 @@ if you want to take the lead for one of them.
 
 We are not looking for out-of-scope contributions, only those that would be considered part of our controlled evolution!
 
-Please read the *Contributing to CodeIgniter* section in the user guide
+Please read the [*Contributing to CodeIgniter*](contributing.md) section in the user guide
```

Comment:
This link won't actually link to the file since they're not in the same folder. Please provide a full URL to the page here on GitHub, so that it will work correctly when viewing on GitHub.

Reply:
Ok will provide complete url of contributing.md